libpages/config/config_v2_test.go

```diff
@@ -0,0 +1,331 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
```

Comment:
Tests are basically the same as `config_v1_test.go`

Reply:
Again, would it be possible to re-factor the tests so you can just pass a config object in for most of them?